lib/features/auth/screens/login_screen.dart

```diff
@@ -277,7 +277,7 @@ class _LoginScreenState extends State<LoginScreen> {
                   const SizedBox(height: 24),
                   if (isSchoolEventFlowEnabled)
                     GivtElevatedSecondaryButton(
-                      text: "I'm at WCA",
+                      text: "Tulsa Changemakers Unite",
```

Comment:
unite and not united? (probably correct but just wanted to double check)

Reply:
According to Figma this is correct :)